frontend/eslint.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Given that we are defining this config in it's own file, should we remove the config from `package.json`: https://github.com/code-for-canada/project-thundercat/blob/master/frontend/package.json#L19